devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -354,6 +365,10 @@ def refresh() {
     delayBetween(cmd, 1000)
 }
 
+def pint() {
```

Comment:
🍺

Reply:
lol Yes, need to push the commit that fixes that. But, I think I'll leave it there for now. ;) It's a good flashback to 2014/2015 ST. :)